content/ngf/installation/installing-ngf/manifests.md

```diff
@@ -53,13 +50,13 @@ Deploying NGINX Gateway Fabric with Kubernetes manifests takes only a few steps.
 
 ---
 
-### 1. Install the Gateway API resources
```

Comment:
Do we not want these numbers in here to show that these steps must be done in order?

Reply:
It's already implied by the the fact they exist in a sequence. It's not a pattern that's used in pages, since the assumption is that the reader goes top to bottom.